### DISCUSSION TITLE
2.0.23.0 missing from Maven repository?

### DISCUSSION THREAD

**Discussion by jaranflaath** [TomRoush/PdfBox-Android#418] [General]
2.0.23.0 has been out for 10 days, but seems to still be missing from the Maven repository. Any news on when it will be available, @TomRoush ?

**Comment by TomRoush** [comment]
It should now be available. For future reference, the artifact should be available through Maven in <4 hours from the release creation. If it isn't available by then, there is either a Maven issue or (more likely) I forgot to finish the manual step to release the artifact into the Maven Central repository

**Reply by jaranflaath** [reply]
Excellent, thanks!
